Modules/_io/fileio.c

```diff
@@ -72,6 +75,7 @@ typedef struct {
     unsigned int closefd : 1;
     char finalizing;
     unsigned int blksize;
+    Py_off_t size_estimated;
```

Comment:
I would prefer to use the same name in the C and Python implementation, I suggest to rename this member to: estimated_size.